internal/data/wallets_test.go

```diff
@@ -533,14 +533,14 @@ func Test_WalletModelSetEnabled(t *testing.T) {
 
 	assert.True(t, wallet.Enabled)
 
-	err = walletModel.SetEnabled(ctx, wallet.ID, false)
+	_, err = walletModel.Update(ctx, wallet.ID, false)
 	require.NoError(t, err)
 
 	wallet, err = walletModel.Get(ctx, wallet.ID)
 	require.NoError(t, err)
 	assert.False(t, wallet.Enabled)
 
-	err = walletModel.SetEnabled(ctx, wallet.ID, true)
+	_, err = walletModel.Update(ctx, wallet.ID, true)
 	require.NoError(t, err)
```

Comment:
💡  Can you also validate that the result of the Update is the same as the Get method?